### ISSUE TITLE
Narrative Needed

### ISSUE THREAD

**Issue by symulation** [hpc-carpentry/hpc-intro#22]
To align further with the standard Carpentries approach the technical content should be wrapped up in a narrative that can serve as a template for participants to imagining themselves fitting into.  I suspect that hpc-novice and hpc-shell should each have their own narrative such that they can be done independently but they could build as long as hpc-novice provided whatever files hpc-shell might produce.

Thoughts?

**Comment by symulation** [comment]
Related to #7 but broader

**Comment by psteinb** [comment]
Yes, yes, yes! You are right on the spot. In [HPC-in-a-day](https://psteinb.github.io/hpc-in-a-day), I did use a narrative which needs some improvement perhaps. I am planning on sending PRs based on that, so some narrative in hpc-shell/hpc-intro would be nice. So I hereby propose to use 'Lola' again. ;)

I like the idea in #7 very much. I think this should go into the [landing page](https://hpc-carpentry.github.io/hpc-intro/) or into [_episodes/00-hpc-intro.md](https://github.com/hpc-carpentry/hpc-intro/blob/gh-pages/_episodes/00-hpc-intro.md). I am more inclined by the latter. This could serve as the basis for the narrative.

**Comment by psteinb** [comment]
+1

**Comment by symulation** [comment]
Yes, into `00-hpc-intro`.  Also into whatever the first episode of `hpc-shell` is.

Perhaps we set up `hpc-shell` as setting up the environment for `hpc-intro`?  Creating some folders, pulling data onto the server, renaming a file or two, etc.  This would allow for one narrative---Lola---but two jumping off points: one where Lola already has the environment set up (straight to `hpc-intro`) and one where she doesn't (`hpc-shell`).  The first thing done in `hpc-intro` if `hpc-shell` was skipped would be to quickly download and unpack a set of files and folders that replicated what was done in `hpc-shell`.  That would be pretty clean, no?

**Comment by psteinb** [comment]
Let's go with this approach. I like it.

**Comment by bkmgit** [comment]
It may be helpful to use software carpentry shell lesson. Most carpentries lessons have setup instructions, so while some knowledge from other lessons can be required, it is helpful to have a fresh setup in each case. If this can be done, this issue can probably be closed.

**Comment by tkphd** [comment]
Alien visitor from a 1-D reality who's never seen a circle before would fit in with (and motivate) the pi example codes. Using a human name might imply  aspersions being cast on those who share it; perhaps Slartibartfast, renowned pangalactic geometer, would be a better namesake.
